typescript-rest/index.ts

```diff
@@ -1,12 +1,16 @@
 
-import {TDConnect,Options} from './src/connect';
-let options:Options = {
+import { TDConnect, Options } from './src/connect';
+let options: Options = {
     path: '/rest/sqlutc',
     scheme: 'http',
+    user: 'root',
+    passwd: 'taosdata',
+    host: "127.0.0.1",
+    port: 6041,
 }
```

Comment:
to compatible with https://docs.taosdata.com/reference/connector/node#%E5%BB%BA%E7%AB%8B%E8%BF%9E%E6%8E%A5